src/ert/gui/model/snapshot.py

```diff
@@ -414,10 +414,10 @@ def _job_data(index: QModelIndex, node: Node, role: int):
         if role == Qt.DisplayRole:
             _, data_name = COLUMNS[NodeType.REAL][index.column()]
             if data_name in [ids.CURRENT_MEMORY_USAGE, ids.MAX_MEMORY_USAGE]:
-                data = node.data.get(ids.DATA)
+                data = node.data
                 _bytes = data.get(data_name) if data else None
                 if _bytes:
-                    return byte_with_unit(_bytes)
+                    return byte_with_unit(float(_bytes))
```

Comment:
what is the type of `_bytes` ? Is this a bug that mypy could have caught?

Reply:
I guess this might have happened when we refactored the snapshot and flattened the data dictionary. The problem was here `data = node.data.get(ids.DATA)`. We check if there is a data section inside of the data section. This would always be None.

Reply:
But we had numbers there before this PR, just not with units (?)

Reply:
Maybe there is a different rendering somewhere else...
I fixed the initial problem, then I got a TypeError because it tried running `>=`between a float value and a str, so it never got to that point before.

Reply:
Ahh, I see what happened now. This if block does not return anything if _bytes was None. It then hit the default return statement at the end of `if role == Qt.DisplayRole:` ... `return node.data.get(data_name)`